kannon/master.py

```diff
@@ -126,16 +126,18 @@ def _create_child_job_object(self, job_name: str, serialized_task: str) -> clien
         ]
         job = deepcopy(self.template_job)
         # replace command
-        assert len(job.spec.template.spec.containers[0].command) == 0, \
-            "command will be replaced by kannon, so you shouldn't set any command and args"
+        original_command = job.spec.template.spec.containers[0].command
+        if original_command:
```

Comment:
unnecessary temporary variable `original_command` ?

Reply:
I think it is better

```python
 assert lob.spec.template.spec.containers[0].command is None, \
            "command will be replaced by kannon, so you shouldn't set any command and args"
```